scripts/create-launchpad-credentials

```diff
@@ -61,12 +61,18 @@ def main():
     print("Now set the following values in environments/development.env, and "
           "make sure to keep them private (including 'chmod 600') as they "
           "allow full access to your Launchpad account:")
+
+    file = open(".env","a")
+
+    file.write("LP_API_URL={}\n".format(service_root.rstrip("/")))
+    file.write("LP_API_USERNAME={}\n".format(lp.me.name))
+    file.write("LP_API_CONSUMER_KEY='{}'\n".format(creds.consumer.key))
+    file.write("LP_API_TOKEN={}\n".format(creds.access_token.key))
+    file.write("LP_API_TOKEN_SECRET={}\n".format(creds.access_token.secret))
+
+    file.close()
     print()
-    print("LP_API_URL={}".format(service_root.rstrip("/")))
-    print("LP_API_USERNAME={}".format(lp.me.name))
-    print("LP_API_CONSUMER_KEY='{}'".format(creds.consumer.key))
-    print("LP_API_TOKEN={}".format(creds.access_token.key))
-    print("LP_API_TOKEN_SECRET={}".format(creds.access_token.secret))
+    print(".env written.")
```

Comment:
Following this change, the output no longer flows very well.  Something like the following will be written to standard output:

  Now set the following values in environments/development.env, and make sure to keep them private (including 'chmod 600') as they allow full access to your Launchpad account:
  .env written.

I'd suggest, instead of this change, adding a `-o`/`--output` option to this program: it already uses `argparse`, so that should be straightforward.  If that option is missing, print the credentials to standard output as before.  If that option is given, omit the "Now set ..." message and write to the requested output file.  Then change `setup.sh` to pass a `-o` option.

(This will probably be easier if you factor the credentials-writing part out to a new function that takes the relevant credentials and the target file object as arguments.)